src/HTMLMesh.js

```diff
@@ -203,7 +223,14 @@ function html2canvas( element ) {
 			context.font = style.fontWeight + ' ' + style.fontSize + ' ' + style.fontFamily;
 			context.textBaseline = 'top';
 			context.fillStyle = style.color;
-			context.fillText( string, x, y + parseFloat( style.fontSize ) * 0.1 );
+			if ( !maxWidth ) {
+				context.fillText( string, x, y + parseFloat( style.fontSize ) * 0.1 );
+			} else {
+				const lines = getLines(context, string, maxWidth);
+				lines.forEach(function(line, i) {
+					context.fillText( line, x, y + parseFloat( style.fontSize ) * 0.1 + i * parseFloat( style.fontSize ) * 1.3 );
```

Comment:
I actually have no idea why I need 1.3 here. I also don't understand why we have the 0.1 here. Is this related to line-height maybe? 0.1 above, 0.1 below + 0.1 between lines?

Reply:
It's to vertically centre the text